src/core/edgeNetwork/injectProcessWarningsAndErrors.js

```diff
@@ -10,33 +10,21 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
+const MESSAGE_PREFIX = "The server responded with the following";
+
 export default ({ logger }) => {
   /**
-   * Processes warnings and errors from a response object. If warnings are found,
-   * they will be logged. If errors are found, an error will be thrown with information
-   * about the errors received from the server.
+   * Processes warnings and errors from a response object by logging them to the
+   * console.
    * @param {Object} response
-   * @param {Object} logger
    */
   return response => {
-    // Regardless of whether the response had a successful status code,
-    // the response payload may have warnings and errors we still
-    // want to process.
-
-    const warnings = response.getWarnings();
-    const errors = response.getErrors();
-
-    warnings.forEach(warning => {
-      logger.warn(
-        `Warning received from server: [Code ${warning.code}] ${warning.message}`
```

Comment:
This is where we are depending on the old format (warnings and errors had `code` and `message` properties). With the new format, there's a lot of good information in the warning and error objects that Konductor returns so I decided to log the objects themselves, rather than try to morph or condense them into a string of some sort.

Reply:
Fatal errors are handled higher up in this same file where we detect if the status is outside the 2xx range: https://github.com/adobe/alloy/blob/e975b9d4415b32469e78e6f48e38af0c4f521461/src/core/edgeNetwork/injectProcessWarningsAndErrors.js#L22-L23